packages/oracle/src/dialect.ts

```diff
@@ -0,0 +1,215 @@
+// Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved
+
+import type { Sequelize } from '@sequelize/core';
+import { AbstractDialect } from '@sequelize/core';
+import type { SupportableNumericOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/abstract-dialect/dialect.js';
+import { parseCommonConnectionUrlOptions } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/connection-options.js';
+import { createNamedParamBindCollector } from '@sequelize/core/_non-semver-use-at-your-own-risk_/utils/sql.js';
+import {
+  BOOLEAN_CONNECTION_OPTION_NAMES,
+  CONNECTION_OPTION_NAMES,
+  NUMBER_CONNECTION_OPTION_NAMES,
+  STRING_CONNECTION_OPTION_NAMES,
+} from './_internal/connection-options.js';
+import * as DataTypes from './_internal/data-types-overrides';
+import { OracleConnectionManager } from './connection-manager';
+import type { OracleConnectionOptions, oracledbModule } from './connection-manager.js';
+import { OracleQueryGenerator } from './query-generator.js';
+import { OracleQueryInterface } from './query-interface.js';
+import { OracleQuery } from './query.js';
+
+export interface OracleDialectOptions {
+  /**
+   *  The oracledb module to user.
+   */
+  oracledbModule?: oracledbModule;
```

Comment:
you declared this but it's not used by the connection manager

Support for this is optional, so you could remove the option